### ISSUE TITLE
Obscure keystrokes

### ISSUE THREAD

**Issue by MichaelNMaggs** [troyhunt/password-purgatory-api#56]
Does everyone know where the ` and ~ keys are on the keyboard?

"Password must contain exactly one backtick character and exactly one tilde character"

**Comment by ubalklen** [comment]
I suggest "at least one" instead of "exactly one" as it seems more reasonable.

**Comment by troyhunt** [comment]
There is nothing about this project that is intended to be "reasonable" 🤣

**Comment by ubalklen** [comment]
@troyhunt yeah, I thought that myself when writing that word 🤣

Just to clarify, my point is that I think the best error messages are the ones that look deceptively credible instead of outright absurd, so a victim can stay hooked for more time.

**Comment by troyhunt** [comment]
Yep, that's fair, although increasing absurdity is also kinda the point 🙂
